.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: Release
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches:
+      - master
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          persist-credentials: false
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - name: Install dependencies
+        run: rm -rf node_modules && yarn install --frozen-lockfile
```

Comment:
I'd prefer this be a run script in package.json so it's easier for developers to do the same steps when troubleshooting CI things, something like `yarn ci:install` or similar (i..e, a `ci` prefix for ci-related run scripts then name it something reasonable for the CI step?)

Reply:
agreed. I'll update